src/commands/is-disclosed.ts

```diff
@@ -0,0 +1,67 @@
+import { BaseCommand } from "../baseCommand";
+import { Formater, OUTPUT_FORMAT } from "../formater";
+import { confirmedFlag, hiddenPropertyKeyFlag, unikidFlag } from "../utils";
+import { GetPropertyValueCommand } from "./get-property-value";
+
+export class IsDisclosedCommand extends GetPropertyValueCommand {
```

Comment:
You should not extend GetPropertyValueCommand, this command use required flags that we don't need is is-disclosed command.